flows/engine/engine.go

```diff
@@ -55,8 +53,6 @@ func ResumeFlow(env flows.FlowEnvironment, run flows.FlowRun, event flows.Event)
 		return run.Session(), nil
 	}
 
-	initTranslations(run)
```

Comment:
I've moved this initialization into flowrun as it seems like it belongs in there